tests/fixtures/modifiers.js

```diff
@@ -8,13 +8,46 @@ const modifiersFixtures = [
 		'pattern': '(?i:[a-z])',
 		'expected': '(?:[A-Za-z])',
 	},
+	{
+		'pattern': '(?i:[ab])',
+		'expected': '(?:[ABab])',
+	},
+	{
+		'pattern': '(?i:\\u212A)',
+		'matches': ['\u212A'],
+		'nonMatches': ['K', 'k'],
+		'expected': '(?:\\u212A)',
+	},
+	{
+		'pattern': '(?i:\\u212A)',
+		'flags': 'u',
+		'matches': ['K', 'k', '\u212A'],
+		'expected': '(?:[Kk\\u212A])',
+	},
+	{
+		'pattern': '(?i:k)',
+		'flags': 'u',
+		'expected': '(?:[Kk\\u212A])',
+	},
+	{
+		'pattern': '(?i:\\u2C2F)',
```

Comment:
This is also an example that a legacy ES5 engine might not support all case foldings in the Basic Plane, because the U+2C2F is introduced in Unicode 14.